content/defender/index.mdx

```diff
@@ -11,6 +11,17 @@ New sign-ups were disabled on June 30, 2025. Until the final shutdown on July 1,
 
 </Callout>
 
+<Callout type='info' title='Migration Guides Available'>
```

Comment:
Maybe just "Defender Migration Guides" and to to remove "We have comprehensive migration guides to help you transition:" 

You can try to update/remove and check how it looks like